tests/test_formatter.py

```
@@ -1679,17 +1677,24 @@ def test_storage(self):
             "rule fmt_bug_repro:\n"
             f"{TAB * 1}run:\n"
             f'{TAB * 2}if "something nested":\n'
             f"{TAB * 3}pass\n"
             f"{TAB * 2}# Comment gets indented\n"
         )
-        formatter = setup_formatter(snakecode)
-        assert formatter.get_formatted() == snakecode
+        assert setup_formatter(expected).get_formatted() == expected
+        snakecode = (
+            "rule fmt_bug_repro:\n"
+            f" run:\n"
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)